### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by BregornOriginal**
# To-Do List Structure

### In this part of the project the structure of it was created whit Webpack

✅ Create an index.html file and write your HTML markup here. Create an empty To Do List `div` or `ul` element

✅In the index.js there is an array of Objects whit a list of tasks to do

✅ This list of the task are created dynamically in the browser whit JS in his corresponding placeholder

✅Webpack was used in this project for the creation of the files in the dist folder

✅ Bootstrap was used for the styles of the elements

> Technologies used:
>
>HTML, CSS, JS ES6, BOOTSTRAP, WEBPACK, GIT, GITHUB, LINTERS

### CODE REVIEW

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @BregornOriginal, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
- [ ] README file should be descriptive
- all main headers from the README template should be present in a student's repo
- Kindly fill all the neccesary details

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Deepakdanger** [comment]
- [ ] While doing review, you have done some changes, kIndly fix linter issue and update readme with all the necessary details.

**Comment by BregornOriginal** [comment]
> * [ ]  While doing review, you have done some changes, kIndly fix linter issue and update readme with all the necessary details.

@Deepakdanger Hi! Thank you for the review. I'm working on the node_modules, I don't know why there are some folders that are not begin ignored from .gitignore. I will try to fix that now. Thank you!

**Comment by BregornOriginal** [comment]
@Deepakdanger Done! 👍🏼

File: README.md [hunk 1]
```diff
@@ -40,14 +54,13 @@
...
 
 ## Authors
 
 👤 **@BregornOriginal**
 
-- GitHub: [@githubhandle](https://github.com/@BregornOriginal)
-
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
```
Comment (BenMukebo) [inline README.md:66]: - [OPTIONAL] Please update your Readme, currently, some sections are missing such as  

``` ##  License ``` section with the MIT license at the end of your README file to make it more professional. I would also recommend you add this. 😄

**Review by BenMukebo — APPROVED** [review]
### Hi @BregornOriginal, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (BregornOriginal) [reply README.md:66]: Ok @BenMukebo , Actually I wasn't using one License already. But I will add one for the next project and add it! Thank you for the suggestion! 💯

**Pull request merged**